### PR TITLE
Remove DeviceType.SupportedDisease usage from backend

### PR DESCRIPTION
**Pull request by emyl3**
# BACKEND PULL REQUEST

## Related Issue

Resolves: #5322 

## Changes Proposed
- Usages of `DeviceType.SupportedDisease` on the backend end should all be removed
- Functionality should not have been affected

## Additional Information
A separate DB PR will be up once this is merged in to drop the `device_supported_disease` table

## Testing
Deployed on dev7

### CODE REVIEW

**Review by BobanL — APPROVED** [review]
Tested in dev7 and looks good!

File: backend/src/main/resources/graphql/main.graphqls [hunk 1]
```diff
@@ -89,7 +87,7 @@ type DeviceType {
   loincCode: String @deprecated(reason: "loincCode is deprecated. Use supportedDiseaseTestPerformed instead.")
   swabType: String @deprecated(reason: "swabType is deprecated. Use swabTypes instead.")
   swabTypes: [SpecimenType!]!
-  supportedDiseases: [SupportedDisease!]!
+  supportedDiseases: String @deprecated(reason: "supportedDiseases is deprecated. Use supportedDiseaseTestPerformed instead.")
```
[the remark this answers is not in the document]
Comment (emyl3) [reply backend/src/main/resources/graphql/main.graphqls:90]: oo true let me remove it!! thank you!

**Review by BobanL — COMMENTED** [review]
➕

**Review by johanna-skylight — APPROVED** [review]
Tested in dev7 and all looks good!

**Review by BobanL — APPROVED** [review]

**Pull request merged**